testing/scenario_app/lib/src/darwin_system_font.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'channel_util.dart';
+import 'scenario.dart';
+
+/// Tries to draw darwin system font: Cupertino-System-Pro, Cupertino-System-Text
+class DarwinSystemFont extends Scenario {
+  /// Creates the DarwinSystemFont scenario.
+  DarwinSystemFont(super.view);
+
+  // Semi-arbitrary.
+  final double _screenWidth = 700;
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+    final PictureRecorder recorder = PictureRecorder();
+    final Canvas canvas = Canvas(recorder);
+
+    final ParagraphBuilder paragraphBuilderPro =
```

Comment:
Nit: should this be paragraphBuilderDisplay? Technically both fonts can be referred to as "Pro" fonts.